egs/material/s5/local/postprocess_test.sh

```diff
@@ -4,35 +4,34 @@ set -e -o pipefail
 set -o nounset                              # Treat unset variables as an error 
 echo "$0 $@"
 
-test_sets=$1
-tree_dir=$2
-dir=$3
+data=$1
+graph_dir=$2
+decode_dir=$3
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
 
-./cmd.sh                                                                        
-./path.sh                                                                       
-./utils/parse_options.sh
 
 # get recording-level CTMs from the lattice by resolving the overlapping
 # regions
 
-for data in $test_sets; do
-  steps/cleanup/internal/get_ctm.sh --frame-shift 0.03 \
-    data/${data}_hires/ $tree_dir/graph/ \
-    $dir/decode_$data/
+steps/get_ctm_fast.sh --frame-shift 0.03 \
+  data/${data}_hires/ ${graph_dir} \
+  ${decode_dir} ${decode_dir}/score_10/
```

Comment:
Dan previously said that the directory structure was rather strange: "score_${lmwt} implies that it's part of some kind of scoring script (which it isn't).  Is there a good reason for this?  I think it should probably be changed."

Reply:
I see from swbd that score_$lmwt are also directly in $decode_dir